mod/settings.php

```diff
@@ -213,6 +212,7 @@ function settings_post(App $a)
 		check_form_security_token_redirectOnErr('/settings/connectors', 'settings_connectors');
 
 		if (x($_POST, 'general-submit')) {
+			PConfig::set(local_user(), 'social', 'disable_cw', intval($_POST['disable_cw']));
```

Comment:
Concerning the category of the config value: Normally "system" is used for every core setting. Addon settings are using the addon name as category.

Reply:
A few lines below, there is a "ostatus" category, did it used to be an addon? What even is the purpose of the category?

Reply:
Yeah, we hadn't been totally strict with it. Just have a look at these settings:
```
PConfig::set(local_user(), 'system', 'ostatus_autofriend',intval($_POST['snautofollow']));
PConfig::set(local_user(), 'ostatus', 'default_group', $_POST['group-selection']);
PConfig::set(local_user(), 'ostatus', 'legacy_contact', $_POST['legacy_contact']);
```

Reply:
That's what I mean, why not use different category name then?

Reply:
This "ostatus" is the bad example. Problem is that there always could be a "social" addon. So possibly we had to use some special category names for addon settings?

Reply:
What do we want to do here?

Reply:
I'm planning on reverting to `system`.

Reply:
Okay.